src/work-allocation/components/case-reference-field/case-reference-field.component.ts

```diff
@@ -0,0 +1,29 @@
+import {Component, Input, OnChanges} from '@angular/core';
+
+@Component({
+  selector: 'exui-case-reference-field',
+  templateUrl: './case-reference-field.component.html'
+})
+export class CaseReferenceFieldComponent implements OnChanges {
+  /**
+   * The caseReference to use for display purposes.
+   */
+  @Input() public caseReference: string;
+
+  private pHref: string;
+  public ngOnChanges(): void {
+    let href;
+    if (this.caseReference) {
+      // Get rid of the spaces in the caseReference.
+      const caseId = this.caseReference.replace(/\s/g, '');
+      if (caseId) {
+        href = `/cases/case-details/${caseId}`;
```

Comment:
This could be a global constant

Reply:
It could, and I looked for one, but I didn't find a global constant anywhere. It's inline like that throughout the codebase. If it's introduced, I think it would make sense to use it for all `/cases/case-details/` instances.

Reply:
Agree with Uday, the base part of that URL should not be stored in the component, but come from config.

Reply:
Will '/cases/case-details/' change periodically? If yes then the configuration is the correct place for it. If no then it's better as a constant within the application. Why? - Because if we placed all strings into our configuration, we would end up with bloat at two points: 1. Within the .json configuration files, and 2. within the configuration/reference file.

Reply:
I've moved it to `AppConstants`. If we need it to be configurable at some point, that's likely going to need an application-wide change.